src/FocusApp.Client/MauiProgram.cs

```diff
@@ -40,6 +42,8 @@ public static MauiApp CreateMauiApp()
                 .RegisterServices()
                 .RegisterPages();
 
+            builder.Services.AddTransient<ProfilePopupInterface>();
```

Comment:
Since we're expected to have multiple popups throughout the app, and each will be registered as a transient, do you think it would be helpful to put this into a method? It could mimic the RegisteredServices method where it registers each individually, or it could mimic RegisterPages and have a BasePopup class. 

What are your thoughts on these suggestions?

Reply:
I added a BasePopup class as suggested as well as a RegisterPopups function similar to the RegisterPages. Good shout 🙌